ui/app/actions.js

```diff
@@ -231,7 +231,7 @@ function createNewVaultAndRestore (password, seed) {
     dispatch(actions.showLoadingIndication())
     log.debug(`background.createNewVaultAndRestore`)
     background.createNewVaultAndRestore(password, seed, (err) => {
-      dispatch(actions.hideLoadingIndication())
+      // dispatch(actions.hideLoadingIndication())
```

Comment:
Do we really want to comment this out?  What if there is not an error?  Does `showAccountsPage` also hide loading indication?